sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/ier12-file-detail.ts

```diff
@@ -97,7 +97,7 @@ export class IER12FileDetail implements IER12FileLine {
       disbursement.dateSent
         ? record.appendDate(disbursement.dateSent, DATE_FORMAT)
         : record.repeatAppend(SPACE_FILLER, 8);
-      disbursement.disbursementValues.map(
```

Comment:
Why changed to `forEach`?

Reply:
Existing sonar bug at main branch. 
https://sonarcloud.io/component_measures?metric=new_reliability_rating&selected=bcgov_SIMS%3Asources%2Fpackages%2Fbackend%2Flibs%2Fintegrations%2Fsrc%2Finstitution-integration%2Fier12-integration%2Fier12-file-detail.ts&view=list&id=bcgov_SIMS